cs-rin-ru-enhanced-mod.user.js

```diff
@@ -923,16 +991,227 @@ async function colorizeFriends() {
         links.forEach(link => {
             let nickname = link.innerText;
             if (link.classList.contains('quotetitle')) nickname = nickname.substring(0, nickname.length - 7)
-            if (USERNAME === nickname) {
+            if (USERNAME === nickname && options.colorize_friends_me === 1 || USERNAME === nickname && options.colorize_friends_me === 3) {
```

Comment:
This condition can be collapsed into
`USERNAME === nickname && (options.colorize_friends_me === 1 || options.colorize_friends_me === 3)`